src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/troubleshooting/missing-entities-relationships.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: "Troubleshoot OpenTelemetry: Missing entities or relationships"
```

Comment:
This document is limited to 1) limited to service entities and 2) barely addresses relationships in any meaningful way.

A document about entities and relationships is sorely needed. Though, it's strange as a "troubleshooting" doc. We really need a top-level document addressing this subject holistically. Including, service, host, and other entity types that can be synthesized from otel data. Plus how relationships work between these entity types.